backend/fms_core/serializers.py

```diff
@@ -583,36 +583,57 @@ class DatasetSerializer(serializers.ModelSerializer):
     released_status_count = serializers.SerializerMethodField()
     blocked_status_count = serializers.SerializerMethodField()
     latest_release_update = serializers.SerializerMethodField()
-
+    total_readsets = serializers.SerializerMethodField()
     class Meta:
         model = Dataset
-        fields = ("id", "external_project_id", "run_name", "lane", "files", "released_status_count", "blocked_status_count", "latest_release_update", "project_name", "metric_report_url")
+        fields = ("id", "external_project_id", "run_name", "lane", "files", "released_status_count", "blocked_status_count", "latest_release_update", "project_name", "metric_report_url", "total_readsets")
```

Comment:
total_readsets name breaks the typical naming pattern ... we typically use "_count".